packages/radix-vue/src/Pagination/utils.ts

```diff
@@ -41,18 +38,37 @@ export function getRange(currentPage: number, pageCount: number, siblingCount: n
 
     const itemCount = totalPageNumbers - 2 // 2 stands for one ellipsis and either first or last page
 
+    const showLeftEllipsis
+      // default condition
+      = leftSiblingIndex > firstPageIndex + 2
+      // if the current page is towards the end of the list
+      && lastPageIndex - itemCount - firstPageIndex + 1 > 2
+      // if the current page is towards the middle of the list
+      && leftSiblingIndex - firstPageIndex > 2
```

Comment:
Haven't test this out, but the inequality seems like the same as line 43 `leftSiblingIndex > firstPageIndex + 2`

Reply:
Yep, my mistake, I adjusted the condition and added `Math.abs` to all calculations to make sure the conditions are always evaluated properly.